src/tribol/common/Parameters.hpp

```diff
@@ -36,11 +36,16 @@ constexpr integer ANY_MESH = -1;
 enum InterfaceElementType
 {
    UNDEFINED_ELEMENT, ///! Undefined
-   NODE,              ///! 0D node
-   EDGE,              ///! 1D edge
-   FACE,              ///! 2D face
-   CELL,              ///! 3D volume
-   NUM_CONTACT_ELEMENTS = CELL
+//   NODE,              ///! 0D node
+//   EDGE,              ///! 1D edge
+//   FACE,              ///! 2D face
+//   CELL,              ///! 3D volume
+   LINEAR_EDGE,       ///! 1D linear edge 
```

Comment:
This was the part that will likely change when we go to high order, so this is an intermediate step. The order + topology serves only to tell me how many nodes there are. The previous EDGE, FACE, CELL isn't enough because a Triangle and a quadrilateral are both faces. If I use this to tell how many nodes a face has, then I assume (for now) they are linear faces. Otherwise, I'd have to have another enum for the order. Here, I baked all that into one for ease. I think going to high-order will be a big enough refactor that no matter what I do here, we will end up touching and modifying it. Thoughts @ebchin ?